cmake/CMakeLists.txt

```diff
@@ -285,6 +285,7 @@ set_target_properties(onnx_proto PROPERTIES FOLDER "External/ONNX")
 # fix a warning in onnx code we can't do anything about
 if (MSVC)
     target_compile_options(onnx_proto PRIVATE /wd4146) # unary minus operator applied to unsigned type
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -W1 -DEIGEN_HAS_C99_MATH /MP")
```

Comment:
This is necessary for using Eigen erf. Without this compilation fails. You can check SpecialFunctionsImpl.h
Although now I do think this may not be the right place to add this... may be line 104 is a better place.

Reply:
Don't change the warning level globally. /W3 or /W4 is mandatory.  You may disable a specific warning for a specific file or vc project. 

```
-DEIGEN_HAS_C99_MATH
```
By default, Eigen strive to automatically detect and enable langage features at compile-time based on the information provided by the compiler. If it doesn't work, let's find out why. At lease we need a reason and a comment for this flag.

```
/MP
```
It's duplicated with line 301.  Please remove it or merge them.

Reply:
ERF requires C++11 and C99
In case of MSVC C99 needs to be set explicitly